packages/teams-js/src/public/pages.ts

```diff
@@ -113,7 +113,7 @@ export namespace pages {
    * @returns Promise that resolves with the {@link InstanceConfig} object.
    */
   export function getConfig(): Promise<InstanceConfig> {
-    return new Promise<InstanceConfig>((resolve) => {
```

Comment:
**I think** you should make sure you've pulled latest and run `yarn`. These `()`s have been added recently when we updated version of prettier. If you take them out, the next person who changes anything in this file will add them back in?

Reply:
Fixed in the latest commit. 👍